CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+This project adheres to [Semantic Versioning](http://semver.org/).
+
+## [1.0.0] - 2015-05-15
+### Breaking Changes
+- iframe.htm introduces breaking change, to better support IE legacy we have to inject/write within an plain script-tag and not via a script-file. If size reporting in IE8 and IE9 are important to you, an upgrade is advised.
```

Comment:
Should probably mention that ext.js doesn't automatically inject anymore. So it's required to update iframe.html when updating ext